src/System.Management.Automation/utils/PathUtils.cs

```diff
@@ -445,10 +451,23 @@ internal static class EncodingConversion
         internal const string BigEndianUnicode = "bigendianunicode";
         internal const string Ascii = "ascii";
         internal const string Utf8 = "utf8";
+        internal const string Utf8NoBom = "utf8NoBOM";
+        internal const string Utf8Bom = "utf8BOM";
         internal const string Utf7 = "utf7";
         internal const string Utf32 = "utf32";
         internal const string Default = "default";
         internal const string OEM = "oem";
+        internal const string Byte = "byte"; // This will return unicode, but 'byte' is special as it uses unicode encoding without a BOM
```

Comment:
Not knowing the inner workings: How will this return _Unicode_? With `Get-Content`, you get a _byte_ array, and on output  (`Set-Content` / `Add-Content`) you're forced to provide a byte array _as input_ too (however, you constructed that).